internal/ingress/defaults/main.go

```diff
@@ -125,6 +125,11 @@ type Backend struct {
 	// Default: false
 	UsePortInRedirects bool `json:"use-port-in-redirects"`
 
+	// Enables or disables relative redirects. By default nginx uses absolute redirects.
+	// http://nginx.org/en/docs/http/ngx_http_core_module.html#absolute_redirect
+	// Default: false
```

Comment:
If the default current is true, we should keep it that.

Reply:
I could make it all (configmap & annotation) with the name `absolute-redirect` 
But then you would have to add an annotation (or configmap) where you set the value explicitly to `false` to get the behavior. Works both for me. 
First thought was to inverse the logic of `absolute_redirect` to relative-redirects so you have to say `yes` to get the new behavior.

Reply:
so?! leave at it is? resolved?

Reply:
No, just keep it. I understand what you mean: `absolute_redirect` in NGINX defaults to `true`.  So it builds absolute redirects by default. This means the default for `relative-redirects` needs to be `false` as we would otherwise build relative redirects by default which would change the default behavior.